doc/problem/classification.md

```diff
@@ -1,26 +1,45 @@
 Project 5: Classification
 ===============================================
-## Statement of overall problem
+
+ 
+ (including how and why it maps to the Statement of Problem)
+ Exploratory Analysis
+ Data Requirements Description
+ Methodology/ Approach Description
+ Testing Framework Outline
+ Additional Remarks
+ References
+ 
+## Statement of Problem
 
 The mouse style research investigates mouse behaviors of different genes. The researchers hope to gain insight to human behaviors using mouse data, since experimenting directly on human is difficult. One question of interest is on whether genetic affects psychological disorders such as depression, or overeating. 
 
-## Detailed summary of statistical problem
+
+## Statement of statistical problems
 
 The researchers design the experiment as follow: they have 16 different strains of mice, and each strain has 12 almost identical mice in term of genetic. If we observe that each mouse behaves very similarly to its twins, but differently from other strains of mice, we can conclude that genetic affect mouse behaviors. The researchers record the daily activities of each mouse, for example the time it spends eating, drinking, sleeping, and wondering around its habitat. 
 
-## Data Description
+## Exploratory Analysis
+
+We dispose of a labeled data set of 16 different strains of mice. Behavioral features are recorded for each mouse and each day, one example can be the time spent eating or drinking. 
```

Comment:
I am not sure what you mean by dispose.  Also are you sure that you have "labeled data set of 16 different strains of mice?"